beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -461,6 +462,7 @@ impl<AppReqId: ReqId, TSpec: EthSpec> Network<AppReqId, TSpec> {
 
         let mut subscribed_topics: Vec<GossipKind> = vec![];
 
+        // FIXME(sproul): inject bls to execution topic here as appropriate
```

Comment:
I realised that this is probably the wrong place to put optional subscriptions, I think I meant to put this in `beacon_node/network/src/service.rs`. However I'm still not sure if that's the right place for a topic that will eventually be core, but is not core at the moment.

How do we normally handle this? cc @pawanjay176 @divagant-martian :pray:

Reply:
If I understand correctly this topic is not really optional (like attnets and sycnets) so I think this is the right place to subscribe to it. If we get messages on this topic before the fork I think they should fail verification. Maybe Pawan has a different opinion

Reply:
I added `BlsToExecutionChange` to `CORE_TOPICS` but I get:

> Nov 22 03:14:52.419 WARN Failed to subscribe to topic, error: NotAllowed, topic: /eth2/2eef3ec8/bls_to_execution_change/ssz_snappy, service: libp2p, module: lighthouse_network::service:589
Nov 22 03:14:52.419 WARN Could not subscribe to topic, topic: /eth2/2eef3ec8/bls_to_execution_change/ssz_snappy, service: network, module: network::service:693

This is at the limit of my networking knowledge but probably trivial for someone knowledgeable, so I'll wait for assistance :pray: